### ISSUE TITLE
Color contrast for tags and other meta on individual posts is not accessible

### ISSUE THREAD

**Issue by jeremyfelt** [washingtonstateuniversity/education.wsu.edu#5]
Flagged as a color contrast ratio of 4.29:1

Example: https://education.wsu.edu/deansperspectives/
